streams/src/main/java/org/apache/kafka/streams/kstream/JoinWindows.java

```diff
@@ -17,15 +17,14 @@
 
 package org.apache.kafka.streams.kstream;
 
-
-import org.apache.kafka.streams.kstream.internals.TumblingWindow;
+import org.apache.kafka.streams.kstream.internals.TimeWindow;
 
 import java.util.Map;
 
 /**
  * The window specifications used for joins.
```

Comment:
I was waiting for https://github.com/apache/kafka/pull/1250 and intentionally didn't touch javadocs in this PR where I could avoid it.